src/app/App.js

```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled, { createGlobalStyle } from 'styled-components';
 import Header from '../components/Header/Header';
-import AdFeed from '../components/FeedPage/AdFeed.js';
+import ListingFeed from '../components/FeedPage/ListingFeed';
+import CreateListing from '../components/CreateListingPage/CreateListing';
 import Footer from '../components/Footer/Footer';
+const listingsArray = require('./mockListings.json');
 
 const GlobalStyles = createGlobalStyle`
```

Comment:
Maybe you want to store your GlobalStyles in an extra file to have a better overview

Reply:
? :)

Reply:
Good Idea :)